source/install/mattermost-kubernetes-operator.rst

```diff
@@ -180,6 +180,141 @@ The process described below needs to be completed prior to proceeding with the M
   The deployment process can be monitored in the Kubernetes user interface. If errors or issues are experienced, review the Mattermost, Operator, and MySQL logs for guidance including error messages. If remediation is not successful, contact Mattermost customer support for assistance.
 
   Once complete, access your Mattermost instance and confirm that the database has been restored.
+Migrate From Helm Chart
+--------------------------------------------
+
+If you have previosuly installed mattermost as helm chart deployment, you can easily move to an operator installed following the provided steps
+
+1. Create a namespace dedicated fo the operator
+
+.. code-block:: sh
+
+  $ kubectl create ns mattermost-operator
+
+2. Save the secrets from the current installation , if you have extra secrets mounted as a volume save them as well
+
+.. code-block:: sh
+
+  $ kubectl get secrets -n mattermost mattermost-db-secret -o yaml > mattermost-db-secret.yaml
+  $ kubectl get secrets -n mattermost mattermost-license-secret -o yaml > mattermost-license-secret.yaml
+  $ kubectl get secrets -n cert -o yaml > cert.yaml
+
+3. Change the db secret to match operator, the helm used mattermost.dbsecret while the operator uses DB_CONNECTION_STRING
+
+.. code-block:: sh
+
+  $ vim mattermost-db-secret.yaml
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: mattermost-db-secret
+  data:
+    DB_CONNECTION_STRING: "db_connection_string"
```

Comment:
I would suggest calling out explicitly the params that need to be customized, to avoid confusion:
```suggestion
    DB_CONNECTION_STRING: [YOUR_CONNECTION_STRING]
```